snapcraft/internal/remote_build/_launchpad.py

```diff
@@ -436,7 +436,7 @@ def _gitify_repository(self, repo_dir: str) -> Git:
         # Commit files.
         timestamp = datetime.now().strftime("%Y-%m-%d %H:%M")
         git_handler.commit(
-            "snapcraft commit\n\nversion: {}\ntimestamp: {}\n".format(
+            "snapcraft version: {} timestamp: {}".format(
```

Comment:
!r considering you are also using spaces as a separator

Reply:
I took out the timestamp, seemed unnecessary given the commit includes one.

Delta: https://github.com/snapcore/snapcraft/compare/33543056f2b04d51cd4071795bc2f00522c30e39..7761c4efbb40208e8e104c9a14d600b7e8eeb82c

The second force push is just a rebase.